pyvista/utilities/reader.py

```diff
@@ -980,6 +1000,279 @@ class Plot3DMetaReader(BaseReader):
+    def family_array_names(self) -> List[str]:
+        """Return the list of all family array names.
+
+        Returns
+        -------
+        list[str]
+
+        """
+        return [self.reader.GetPointArrayName(i) for i in range(self.number_point_arrays)]
+
+    def enable_all_families(self):
+        """Enable reading all families.
+
+        By default only the 0th family is read.
+
+        Examples
+        --------
+        Read all bases.
+
+        >>> import pyvista
+        >>> from pyvista import examples
+        >>> filename = examples.download_cgns_multi(load=False)
+        >>> reader = pyvista.get_reader(filename)
+        >>> reader.enable_all_families()
+        """
+        self._reader.EnableAllFamilies()
+
+    def disable_all_families(self):
+        """Disable reading all families.
+
+        Examples
+        --------
+        Disable reading all bases.
+
+        >>> import pyvista
+        >>> from pyvista import examples
+        >>> filename = examples.download_cgns_multi(load=False)
+        >>> reader = pyvista.get_reader(filename)
+        >>> reader.disable_all_families()
+        """
+        self._reader.DisableAllFamilies()
+
+    @property
+    def unsteady_pattern(self) -> bool:
+        """Return or set using an unsteady pattern.
+
+        When set to ``True`` (default is ``False``), the reader will try to
+        determine to determine FlowSolution_t nodes to read with a pattern
+        matching This can be useful for unsteady solutions when
+        FlowSolutionPointers are not reliable.
+
+        Examples
+        --------
+        Set reading the unsteady pattern to ``True``.
+
+        >>> import pyvista
+        >>> from pyvista import examples
+        >>> filename = examples.download_cgns_multi(load=False)
+        >>> reader = pyvista.get_reader(filename)
+        >>> reader.unsteady_pattern = True
+        >>> reader.unsteady_pattern
+        True
+
+        """
+        return self._reader.GetUseUnsteadyPattern()
+
+    @unsteady_pattern.setter
+    def unsteady_pattern(self, enabled: bool):
+        self._reader.SetUseUnsteadyPattern(bool(enabled))
+
+    @property
+    def vector_3d(self) -> bool:
+        """Return or set adding an empty dimension to vectors in case of 2D solutions.
+
+        Examples
+        --------
+        Set adding an empty physical dimension to vectors to ``True``.
+
+        >>> import pyvista
+        >>> from pyvista import examples
+        >>> filename = examples.download_cgns_multi(load=False)
+        >>> reader = pyvista.get_reader(filename)
+        >>> reader.3d_vector = True
+        >>> reader.3d_vector
+        True
+
+        """
+        return self._reader.GetUseUnsteadyPattern()
+
+    @vector_3d.setter
+    def vector_3d(self, enabled: bool):
+        self._reader.SetUse3DVector(bool(enabled))
+
+    @property
+    def load_boundary_patch(self) -> bool:
+        """Return or set loading boundary patches.
+
+        Default is ``False``.
```

Comment:
OpenFOAM sets a similar kind of default to true.  Should we do that here? Otherwise the surface data won't be read in by default.

Reply:
Agreed and enabled in `__init__`